src/loaders/workspace_downloader/workspace_downloader.py

```diff
@@ -10,6 +10,10 @@
 required named arguments:
   --workspace_id WORKSPACE_ID
                         Workspace addressed by the permanent ID
+  --collection COLLECTION
+                        Create a collection and link in data to that colleciton from the overall workspace source data dir
+  --source_version SOURCE_VERSION
+                        Create a source version and link in data to that colleciton from the overall workspace source data dir
```

Comment:
These can be optional - if there's no collection given, don't make a collection. If one is required, both are required though

Reply:
Right

Reply:
Updated!

Reply:
:+1: